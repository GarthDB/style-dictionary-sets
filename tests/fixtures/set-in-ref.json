{
  "component": {
    "width": { "value": "{size}" }
  },
  "size": {
    "sets": {
      "mobile": { "value": "15px" },
      "desktop": { "value": "12px" }
    }
  }
}
