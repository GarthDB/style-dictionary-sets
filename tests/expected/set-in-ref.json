{
  "component": {
    "width": {
      "ref": "{size}",
      "sets": {
        "mobile": {
          "value": "15px"
        },
        "desktop": {
          "value": "12px"
        }
      }
    }
  },
  "size": {
    "sets": {
      "mobile": {
        "value": "15px"
      },
      "desktop": {
        "value": "12px"
      }
    }
  }
}
